wg21_paper_tracker/management/commands/run_wg21_paper_tracker.py

```
@@ -0,0 +1,92 @@
+                project_id = getattr(settings, "GCP_PROJECT_ID", None)
+                location = getattr(settings, "GCP_LOCATION", "us-central1")
+                job_name = getattr(settings, "WG21_CLOUD_RUN_JOB_NAME", None)
+                bucket = getattr(settings, "WG21_GCS_BUCKET", None)
+
+                if project_id and job_name and bucket:
+                    try:
+                        trigger_cloud_run_job(project_id, location, job_name)
+                        logger.info(
+                            "Successfully triggered Cloud Run job %s.", job_name
+                        )
+                    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)